.travis.yml

```diff
@@ -20,7 +27,19 @@ before_script:
   - cp sites/public/.env.template sites/public/.env
   - cp sites/partners/.env.template sites/partners/.env
   - cp backend/core/.env.template backend/core/.env
-  - yarn install:all
+
+defaults: &defaults
+  script:
+    - yarn cypress version
```

Comment:
Does this do anything beyond spitting out the cypress version?

Reply:
Also it looks like it's called twice?

Reply:
Ah you caught me in the middle of still trying to debug why it's not working. I thought I had it working, but it's still failing.